README.md

```diff
@@ -58,12 +58,12 @@ Thank you for your interest in contributing, you are awesome!!!! yet in order to
 
 ### API Spec
 
-Please follow: api-spec.md document.
+Please follow: api-spec.md document (TODO).
```

Comment:
Should be referring to this document: https://github.com/Midburn/burner-games/blob/master/technical-design.md